source/opt/ccp_pass.h

```diff
@@ -67,6 +67,13 @@ class CCPPass : public MemPass {
   // otherwise.
   bool ReplaceValues();
 
+  // Marks |instr| as varying by registering a varying value for its result
+  // into the |values_| table. Returns SSAPropagator::kVarying.
+  SSAPropagator::PropStatus MarkInstructionVarying(ir::Instruction* instr);
+
+  // Returns true if |id| is a varying value.
+  bool IsVaryingValue(uint32_t id) const;
+
   // Constant manager for the parent IR context.  Used to record new constants
   // generated during propagation.
   analysis::ConstantManager* const_mgr_;
```

Comment:
You must also update the comment for the values_ member. Should say that an SSA id can map to the kVaryingValueSentinelId to indicate that the id has been proven to be varying.

Reply:
Done.